design/protocol-changes/object-diffusion.tex

```diff
@@ -104,18 +106,17 @@ \subsection{Protocol messages}
       The client asks for up to \argfont{req} new object ids and acknowledges \argfont{ack} old ids.
       The server will block until new objects are available.
 
-\item [\MsgReplyObjIds{} {($[(\argfont{id}, \argfont{objectMetadata})]$)}]
-      The server replies with a list of available objects, represented by their ids.
-      The list contains pairs of object ids and some metadata about them.
-      In the blocking case, the reply is guaranteed to contain at least one object id.
-      In the non-blocking case, the reply may contain an empty list.
+\item [\MsgReplyObjIds{} {$(\argfont{objectIds})$}]
+      The server replies with the ids of its available objects.
+      In the blocking case, the reply is guaranteed to contain enough information to build at least one object id with \argfont{responseToIds}.
+      In the non-blocking case, the reply may not contain any actual data.
 
 \item [\MsgRequestObjs{} {($[\argfont{id}]$)}]
       The client requests objects by sending a list of object ids.
       The total size of the corresponding objects MUST not be bigger than the size limit in bytes.
       \niols{Or maybe a bit less; should we take into account the size of the boilerplate?}
 
-\item [\MsgReplyObjs{} {($[\argfont{object}]$})]
+\item [\MsgReplyObjs{} {($[\argfont{object}]$)}]
```

Comment:
In fact, I'd even go for `$( ... )$` to get the math parentheses. But then we should probably go for `$\MsgReplyObjs{} ( ... )$` with a macro to create the right spacing after `\MsgReplyObjs` with eg. `\mathop`. Let's no go there :p